storm-core/src/jvm/org/apache/storm/executor/Executor.java

```diff
@@ -204,8 +204,8 @@ public static Executor mkExecutor(Map workerData, List<Long> executorId, Map<Str
                 throw Utils.wrapInRuntime(ex);
             }
         }
-        executor.init(idToTask);
 
+        executor.idToTask = idToTask;
```

Comment:
@HeartSaVioR Sorry, missed that. This code looks convoluted because it is ported from clojure. I am fine with that for now as these PRs are more about poring from clojure to Java. We can address these later.